exec/java-exec/src/main/java/org/apache/drill/exec/physical/config/ExternalSort.java

```diff
@@ -49,12 +49,19 @@ public int getOperatorType() {
     return CoreOperatorType.EXTERNAL_SORT_VALUE;
   }
 
-  // Set here, rather than the base class, because this is the only
-  // operator, at present, that makes use of the maximum allocation.
-  // Remove this, in favor of the base class version, when Drill
-  // sets the memory allocation for all operators.
-
+  /**
```

Comment:
But still need a local comment to be clear about this code .... so does not matter much.